### DISCUSSION TITLE
WebDriverIO project saved in network drive doesn't run

### DISCUSSION THREAD

**Discussion by akbarkarim** [webdriverio/webdriverio#8014] [Q&A]
We are keeping our WebDriverIO based automated test suite in network drive, so that my team mates can also access and run the scripts when needed.  However when we try to run the script from the shared location, it doesn't trigger and takes a long time without any further information, and eventually we have to abort it.

Is there any special config to be done to work in this kind of set up?

**Comment by portyu9** [comment]
Hi @akbarkarim 

There is no special WebdriverIO option required for running from a network drive.

However, I would strongly recommend **not executing the Node/WebdriverIO project directly from a shared network filesystem**. Node has to perform a large number of filesystem operations while resolving modules and loading `node_modules`, and network filesystems can make that extremely slow or expose filesystem/permission behavior that local tooling doesn't expect.

A better setup is:

1. Put the test project in Git.
2. Let each teammate clone/pull it locally.
3. Run `npm install` / `npm ci` locally.
4. Execute WDIO from the local checkout.

If you need to diagnose the existing setup, run it once with WDIO debug logging and compare the same project copied to a local directory. If it works locally but hangs from the share, that isolates the problem to the filesystem/environment rather than the WDIO tests themselves.

Since no logs or error output were included in the discussion, I don't think there is enough evidence to identify a more specific root cause.

I reviewed the full thread; there were no replies, screenshots, or attachments.

Please mark this answer as accepted if it helped, thank you.
